tests/onnx/data/reference_graphs/original_nncf_graph/synthetic/output_with_no_parents_model.dot

```diff
@@ -0,0 +1,8 @@
+strict digraph  {
+"0 Conv1" [id=0, type=Conv];
+"1 nncf_model_input_0" [id=1, type=nncf_model_input];
+"2 nncf_model_output_0" [id=2, type=nncf_model_output];
+"3 nncf_model_output_1" [id=3, type=nncf_model_output];
```

Comment:
It looks like the graph is not connected. Let's discuss offline what impact on algorithms.

Reply:
The logic was updated. Outputs with no parent node is not added to the NNCFGraph.